src/main/java/org/takes/rq/multipart/RqMtBase.java

```diff
@@ -135,7 +136,9 @@ public RqMtBase(final Request req) throws IOException {
     @Override
     public Iterable<Request> part(final CharSequence name) {
         final List<Request> values = this.map.getOrDefault(
-            new EnglishLowerCase(name.toString()).string(),
+            new UncheckedText(
+                new Lowered(name.toString())
```

Comment:
@fanifieiev Isn't `name` already `String`? Why use `toString()` here?

Reply:
@fabriciofx Because key is CharSequence